### DISCUSSION TITLE
Capstone DA [Flask Ui] - Group by Data Unmatch dengan Contoh Task 6

### DISCUSSION THREAD

**Discussion by mohammadbaguskurniawan** [teamalgoritma/community#655] [Q&A] [answered]
Hai, kenapa data hasil wrangling data dengan group by saya tidak sesuai dengan contoh yang ada?

![image](https://github.com/teamalgoritma/community/assets/76095978/164b5cd4-74c2-456e-b2ef-d096223ff042)

Thank You

**Comment by nadyadtm** [comment]
Dear Pak @mohammadbaguskurniawan 

Terima kasih Pak untuk pertanyaannya, akan kami coba lihat terlebih dahulu

**Comment by nadyadtm** [comment, marked as answer]
Halo @mohammadbaguskurniawan , untuk menjaga agar forum diskusi berjalan kondusif, kami izin melakukan close pada diskusi ini karena sudah mengumpulkan capstone dan jawabannya sudah tersolved. Semoga dapat dipahami
